.cursor/skills/teapie/references/variables-functions.md

```
@@ -0,0 +1,224 @@
+
+**Notes:**
+
+- Arguments are whitespace-separated (maximum two per function)
+- Use quotes for values with spaces: `{{$now "yyyy-MM-dd HH:mm"}}`
+- Both single and double quotes are supported
+- Can mix with variables: `{{$add {{MyNumber}} 2}}`
+
+### Built-in Functions
+
+| Name | Signature | Description | Example |
+|------|-----------|-------------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)